pages/docs/reason-compiler/latest/decorators.mdx

```diff
@@ -808,7 +848,49 @@ log(404, "NOT_FOUND", "this", "page", "is", "not", "found");
 ```
 
 ## `[@bs.inline]`
-TODO
+
+Especially for library authors, `bs.inline` could be interesting. It tells the compiler explicitly to inline a constant directly. 
+
+Suppose you have the following binding for the node environment variable:
+
+```reason
+/* NodeEnv.re */
+[@bs.val] [@bs.scope "process.env"] external env: string = "NODE_ENV";
+
+// Now you can define possible values for it with `bs.inline`:
+[@bs.inline]
+let development = "development";
+
+[@bs.inline]
+let production = "production";
+```
+
+Invoke them in another module: 
+
+```reason
+if (NodeEnv.env === NodeEnv.development) {
+  Js.log("development");
+}
+```
+
+the corresponding value gets inlined directly (in this case the string `development`) in the JS code:
+
+```js
+if (process.env.NODE_ENV === "development") {
+  console.log("development");
+}
+```
+
 
 ## `[@bs.meth]`
-TODO
+
+To call a method of a `Js.t` field, use `bs.meth` to avoid issues with a curried object where the reference to `this` may be incorrect.
```

Comment:
I don't understand this sentence... does this mean that if you want to use a function which needs to bind to the object's `this` context, you need to use `bs.meth`?

Maybe it helps to see a real world JS object in JavaScript code first to explain how this would be applied?